flink-cdc-connect/flink-cdc-source-connectors/flink-connector-oracle-cdc/src/main/java/org/apache/flink/cdc/connectors/oracle/source/utils/OracleSchema.java

```diff
@@ -42,10 +42,7 @@ public OracleSchema() {
         this.schemasByTableId = new HashMap<>();
     }
 
-    /**
-     * Gets table schema for the given table path. It will request to MySQL server by running `SHOW
-     * CREATE TABLE` if cache missed.
-     */
+    /** Gets table schema for the given table path. It will request to Oracle server by running */
```

Comment:
Is this sentence complete?

Reply:
![image](https://github.com/user-attachments/assets/bdd9f29a-6d65-458d-8c97-ddd30ba85f5b)

![image](https://github.com/user-attachments/assets/e69a904e-ef54-48dd-96ed-4dd2997d12db)

It is complete, because I found that it was not obtained by executing sql, it was obtained by java.sql.connect #getMetaData

Reply:
Gets the table schema for a given table path. It will request from the Oracle server via ` java.sql.connect #getMetaData`

Reply:
Or is it ok if I change it to this

Reply:
Since there's no server request at all, what about removing `It will request to Oracle server by running`?

Reply:
`/** Gets table schema for the given table path. */`

 Is it OK to change it like this